packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -922,19 +923,25 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             maxOpsSinceLastSummary,
             summarizerClientElectionEnabled,
         );
-        // Create the SummaryManager and mark the initial state
-        this.summaryManager = new SummaryManager(
-            context,
-            this.summarizerClientElection,
-            this.runtimeOptions.summaryOptions.generateSummaries !== false,
-            this.logger,
-            this.runtimeOptions.summaryOptions.initialSummarizerDelayMs,
-            this.runtimeOptions.summaryOptions.summarizerOptions,
-        );
-
-        if (this.connected) {
-            // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            this.summaryManager.setConnected(this.context.clientId!);
+        // Only create a SummaryManager if summaries are enabled and we are not the summarizer client
+        if (this.runtimeOptions.summaryOptions.generateSummaries === false) {
+            this.logger.sendTelemetryEvent({ eventName: "SummariesDisabled" });
```

Comment:
This will fire for every client when they are disabled. I think it would be more valuable to only fire when it is elected?

Also below could be else?

Reply:
I'm not sure - here it will give a reliable report of who is enabling the configuration (which might be harder to detect if it doesn't always fire).  Firing only-when-elected will highlight an inefficiency in scenarios where SOME clients have summaries disabled (is that a real scenario?), but is meaningless in scenarios where ALL clients have summaries disabled.

Reply:
I think firing only when elected is a more accurate depiction of the problem. Right now, I agree that it's pretty much always the case that all clients will have it disabled or none. But in that case, one (write) client will still always be elected, and that client will log the error.

As it is now, all clients will be firing this event when they connect. So we will be sending a lot of excess events.

If someone did start using the flag only for certain clients (maybe readonly?), then it would fire excess logs for them.